samples/crypto/psa_its/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _crypto_psa_its:
+
+Crypto: PSA ITS
+##############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample shows how to use the Platform Security Architecture (PSA) Internal Trusted Storage (ITS) APIs.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
```

Comment:
the txt doesn't match the other crypto sample as well. I would propose to base this on the other crypto samples.

Reply:
fixed. based on persistent_key_usage now.